client/commands/v2/pysa_server.py

```diff
@@ -491,23 +230,15 @@ async def run_persistent(
 
             client_capabilities = initialize_result.client_capabilities
             LOG.debug(f"Client capabilities: {client_capabilities}")
-            initial_server_state = ServerState()
             server = PysaServer(
                 input_channel=stdin,
                 output_channel=stdout,
                 client_capabilities=client_capabilities,
-                state=initial_server_state,
-                pyre_manager=connection.BackgroundTaskManager(
-                    PysaServerHandler(
-                        binary_location=binary_location,
-                        server_identifier=server_identifier,
-                        pyre_arguments=pysa_arguments,
-                        client_output_channel=stdout,
-                        server_state=initial_server_state,
-                    )
-                ),
+                binary_location=binary_location,
+                server_identifier=server_identifier,
+                pyre_arguments=pysa_arguments,
             )
-            return await server.run()
+            return await server._run()
```

Comment:
`_` methods are considered private in Python. If we intend to call it from outside the class, the method should stay as `run` rather than `_run`